wporg-gp-translation-events.php

```diff
@@ -24,6 +24,7 @@
 use GP;
 use WP_Post;
 use WP_Query;
+use function Symfony\Component\VarDumper\Dumper\esc;
```

Comment:
I guess this is for debugging purposes.

Reply:
Yes, I think it was added by the IDE, because it is unused, and I don't use it to debug. I am going to remove it in a new commit. Thank you!